misc/acrn-config/board_config/new_board_kconfig.py

```diff
@@ -115,9 +122,11 @@ def generate_file(config):
     ram_range = get_ram_range()
 
     # reseve 16M memory for hv sbuf, ramoops, etc.
-    reserved_ram = 0xf00000
+    reserved_ram = 0x1000000
+    # hard code hv ram start location: 256M
```

Comment:
this comment is confusing, since the final hv start address could be changed later.
Could you modify the comment to be more accurate?

Reply:
Done!